release-notes.html.md.erb

```diff
@@ -1,709 +1,38 @@
 ---
-title: Healthwatch v2.2 Release Notes
+title: Healthwatch v2.3 Release Notes
 owner: Healthwatch
 ---
 
-This topic contains release notes for Healthwatch&trade; for VMware Tanzu^&reg;^ v2.2 (Healthwatch).
+This topic contains release notes for Healthwatch&trade; for VMware Tanzu^&reg;^ v2.3 (Healthwatch).
 
-For information about the risks and limitations of Healthwatch v2.2, see [Assumed Risks of Using Healthwatch v2.2](index.html#assumed-risks) and [Healthwatch
-v2.2 Limitations](index.html#healthwatch-limitations) in _Healthwatch for VMware Tanzu_.
+For information about the risks and limitations of Healthwatch v2.3, see [Assumed Risks of Using Healthwatch v2.3](index.html#assumed-risks) and [Healthwatch
+v2.3 Limitations](index.html#healthwatch-limitations) in _Healthwatch for VMware Tanzu_.
 
 ## <a id='releases'></a> Releases
 
-### <a id='2-2-9'></a> v2.2.9
+### <a id='2-3-1'></a> v2.3.1
 
-**Release Date:** October 12, 2023
+**Release Date:** April 10, 2024
 
-* **[Breaking Change]** Healthwatch now requires the use of [system-metrics-agent](https://github.com/cloudfoundry/system-metrics-release/blob/main/docs/system-metrics-agent.md)
-processes to gather "system" metrics from BOSH deployed VMs.  Please make sure to enable `Enable System Metrics`
-in the `Director Config` tab of the BOSH Director tile. Ensure that `Apply Changes` is run on all tiles and that
-all VMs deployed via service brokers are upgraded prior to upgrading Healthwatch. The impact of not performing this
-action is that Healthwatch dashboards will fail to populate with metrics about VM health, cpu, memory, disk and other
-statistics. Alerts may also be affected. Any custom dashboards that query "system" metrics with an `origin` label value
-of `bosh-system-metrics-forwarder` will need to be updated to use `system-metrics-agent` as the `origin` label value.
+* **[Breaking Change]** Changes might be observed in Healthwatch metrics on switching to OTel collector for metric collection. All such changes have been described in
+this section. Metrics that were coming from Firehose-specific components like reverse log proxy will not be available after enabling OTel collector. Currently the OTel
+collector is collocated on TAS VMs and will only collect metrics from these VMs. As a result of this, metrics from other tiles will not be collected by the OTel collector. The
+prometheus exporter in OTel collector does not allow leading underscore, except in private metrics. So the OTel collector is removing any leading underscores in metric
+names. Also SVM forwarder VM will not work with otel and will have to be switched off if it is switched on.
 
-* **[Feature]** Added support of VMware Tanzu Kubernetes Grid Integrated Edition v1.16.
+* **[Feature]** Healthwatch 2.3.1 has support for using OTel collectors (instead of Firehose) for collecting metrics. Healthwatch automatically switches to collecting metrics
```

Comment:
Text suggestion:
**[Feature]** Healthwatch 2.3.1 supports using the OpenTelemetry collector (instead of Firehose) to collect metrics. Healthwatch automatically collects metrics
from the OpenTelemetry collector when the option on TAS for Enabling Firehose v1/v2 is disabled.